02.wo_Defects/uninit_pointer.c

```diff
@@ -262,7 +262,8 @@ void uninit_pointer_012 ()
 	}
 	for(i=0; i<10; i++)
 	{
-		arr[i] = ++fptr[i]; /*Tool should not detect this line as error*/ /*No ERROR:Uninitialized pointer*/
+		++fptr[i];
+		arr[i] = fptr[i]; /*Tool should not detect this line as error*/ /*No ERROR:Uninitialized pointer*/
```

Comment:
fptr and arr are aliases of each other at this point so we actually have a side effect and value computation that are unsequenced from each other here.